app/javascript/components/track/Approaches.tsx

```diff
@@ -5,7 +5,7 @@ import {
   DiggingDeeper,
 } from './approaches-elements'
 
-export function Approaches(): JSX.Element {
+export function Approaches({ data }: { data: any }): JSX.Element {
```

Comment:
I assume that you'll add a proper type later?

Reply:
yes